br/backup-and-restore-storages.md

```diff
@@ -156,7 +157,7 @@ It is recommended that you configure access to S3 using either of the following
 
 - Method 1: Specify the access key
 
-    If you specify an access key and a secret access key in the URL, authentication is performed using the specified access key and secret access key. Besides specifying the key in the URL, the following methods are also supported:
+    If you specify an access key and a secret access key in the URI, authentication is performed using the specified access key and secret access key. Besides specifying the key in the URI, the following methods are also supported:
```

Comment:
Does the Chinese doc version need the same update?

Reply:
yes